lib/components/Navigation/styles.ts

```diff
@@ -0,0 +1,222 @@
+import { css } from "@emotion/css";
+import breakpoints from "../../theme/breakpoints";
+
+export const navMain_DesktopLink = css`
+  text-decoration: none;
+  padding: 0px 30px;
+  font-weight: 500;
+  color: var(--surface-05) !important;
+  display: flex;
+  align-items: center;
+  font-size: var(--font-size);
+
+  &:hover {
+    color: var(--surface) !important;
+  }
+
+  &[data-active="true"] {
+    color: var(--surface) !important;
+  }
+`;
+
+export const navMain_Mobile = css`
+  position: absolute;
+  top: 0px;
+  right: 0px;
+  width: 100%;
+  height: 100%;
+  overflow-y: scroll;
+  background-color: var(--white);
+  box-shadow: rgba(0, 0, 0, 0.1) 0px 0px 4px 4px;
+  transform: translate3d(0px, 0px, 0px);
+  transition: transform 0.5s ease 0s;
+  padding-top: 64px;
+  display: flex;
+  ${breakpoints.medium} {
+    display: none;
+  }
+`;
+
+export const navMain_MobileClosed = css`
+  position: absolute;
+  top: 0px;
+  right: 0px;
+  width: 100%;
+  height: 100%;
+  overflow-y: scroll;
+  background-color: var(--white);
+  transform: translate3d(100%, 0px, 0px);
+  transition: transform 0.5s ease 0s;
+  padding-top: 64px;
+  display: flex;
+  ${breakpoints.medium} {
+    display: none;
+  }
+`;
+
+export const navMain_MobileList = css`
+  padding: 0px;
+  padding-top: 20px;
+  list-style: none;
+  margin: 0px auto;
+`;
+
+export const navMain_MobileItem = css`
+  padding: 12px 24px;
+`;
+
+export const navMain_MobileLink = css`
+  display: block;
+  width: 100%;
+`;
+
+export const navMain_MobileItemInner = css`
+  font-size: 3.75rem;
+  font-weight: 500;
+  color: var(--surface-02);
+  padding: 4px 0px 0px;
+  display: flex;
+  align-items: center;
+
+  &:hover {
+    color: var(--surface-04);
+  }
+`;
+
+export const navMain_MobileItemInnerNumber = css`
+  font-size: 2rem;
+  font-weight: 600;
+  color: var(--surface-05);
+  margin-right: 20px;
```

Comment:
since we are making progress on translation system we have to be ready for RTL languages.

`margin-inline-start` will work on both RTL and LTR

But my personal fav is, where possible, to have parent div space children using grid or flex `gap`

Reply:
in this special case I think it is ok as it targets the number next to the main nav link on the menu item
Will keep in mind flex grid for the future